pages/Export.md

```diff
@@ -1,4 +1,66 @@
 type:: [[Feature]]
 platforms:: [[All Platforms]]
+description:: The Logseq Export feature allows users to export their data in various formats, including text, HTML, OPML, PNG, EDN, JSON, and standard Markdown. It supports exporting text selections, blocks, individual pages, or the entire graph with customizable options.
 
-- TODO Document feature, page and graph level #docs
\ No newline at end of file
+- ## Introduction
+	- The Logseq export feature enables users to export selection, a single block or a page, or the entire graph into various formats, such as plain-text, OPML, HTML, or PNG. This feature is designed to enhance the user experience by providing a convenient way to share, backup, or migrate Logseq data to other platforms and applications.
+	- Purpose and benefits include improving compatibility with other tools, simplifying data backup and sharing, and providing multiple export options to suit user requirements.
+- ## Purpose and Benefits
+	- **Cross-platform Compatibility**: The export feature allows for better integration with other tools by supporting common file formats such as plain text, OPML, HTML, EDN, JSON, and standard Markdown. This cross-platform compatibility makes it easier to migrate data or use your Logseq content in different applications.
+	- **Data Backup**: The export feature provides an efficient way to create backups of your Logseq graph. Regular backups can help prevent data loss and ensure data integrity.
+	- **Flexible Sharing Options**: Exporting allows you to share your Logseq content with others in a format they can easily view or import into other tools. It also enables you to present your content in various formats suitable for different contexts, such as exporting a block to an image.
+	- **Customization**: The export feature offers numerous options to customize the exported content, such as defining the  scope of export {{cloze selection, a single block, a page, or the entire graph}}, choosing the export format, and selecting specific options related to each format. This flexibility allows users to tailor the export process to their specific needs.
+- ## Export Scopes
```

Comment:
Feature pages have a specific format, https://github.com/logseq/docs/blob/master/CONTRIBUTING.md#documenting-a-feature, and we should keep with it. I'll move content around to fit the format

Reply:
@cldwalker got it will make sure to stick to it in the future.  This feature is a bit dense with information initially I ended u with everything under the usage section.

Thanks for fixing it :D